.github/workflows/test.yml

```diff
@@ -78,18 +100,17 @@ jobs:
           env_vars: OS, python
 
   test-integration:
+    needs: prepare-matrix
     timeout-minutes: 20
     runs-on: ${{ matrix.os }}
+    if: >-
+      !(
+        fromJson(needs.prepare-matrix.outputs.matrix-integration).python-version == '3.8' &&
+        fromJson(needs.prepare-matrix.outputs.matrix-integration).borg-version == '2.0.0b5'
+      )
```

Comment:
I theory this will work the same way as an `exclude` but for dynamically generated matrices, the only difference being that the combinations will still show up as a job, they should just not execute.